lib/community.js

```diff
@@ -409,7 +409,14 @@ exports.all = {
         quote: 'Coming from an Erlang thinking, was looking for a fast, reliable and robust framework, that also would be accessible by normal programmers (only 1% of programmers write in Erlang, it\'s for sure an anomaly). Looking and testing more than 30 frameworks, we noticed that everything was built in Express, a cool framework for websites, but not built from scratch to be a "java beans" killer, I mean, an enterprise framework. So I found myself comparing hapi against all the others, and in fact, taking all the others frameworks on one hand, they will not be able to reach what hapi reaches, since it was built from the ground up for the enterprise development.',
         person: 'Henry Hazan, CEO'
     },
-    'Walmart': {
+    'The Freecycle Network': {
+        url: 'http://freecycle.org',
+        logo: 'logo-freecycle.jpg',
+        width: 150,
+        quote: 'Hapi was impressive for its community support, reliability, and rate of adoption by other organizations and developers. As a scrappy, low-budget, non-profit charity, we want to leverage our volunteer base, which includes developers who can plug in and help out with building and modifying our web applications. Hapi seems like the future that a lot of Node coders are going to know.',
```

Comment:
kind of a nit, but hapi is always lowercase. even at the beginning of a sentence.

neat project :)